pkg/gohai/memory/memory_linux.go

```diff
@@ -31,20 +31,30 @@ func parseMemoryInfo(reader io.Reader) (totalBytes utils.Value[uint64], swapTota
 	totalBytes = utils.NewErrorValue[uint64](fmt.Errorf("'MemTotal' not found in /proc/meminfo"))
 	swapTotalKb = utils.NewErrorValue[uint64](fmt.Errorf("'SwapTotal' not found in /proc/meminfo"))
 	for _, line := range lines {
-		pair := regexp.MustCompile(": +").Split(line, 2)
-		values := regexp.MustCompile(" +").Split(pair[1], 2)
+		key, valUnit, found := strings.Cut(line, ":")
+		if !found {
+			continue
+		}
```

Comment:
I checked the implementation in the Linux Kernel and it seems like every line should contain a `:`, however it seems weird to me to log a warning because a line I don't need to read doesn't have the format I thought it should have
For all I know some specific platforms could add lines without `:`, or one day the format might be changed, here all I care about are the specific lines I want to read...

What do you think ?

Reply:
I'm mainly worried about us not seeing issues with the parsing surfaced in the only way we can (flares/logs/etc). If you don't believe it's a problem we need to worry about, it should be ok.

Reply:
That's fair, we can always remove the log line if someone complains it's a false positive anyway